utility/src/test/java/org/oppia/android/util/parser/html/MathTagHandlerTest.kt

```diff
@@ -141,6 +150,9 @@ class MathTagHandlerTest {
   }
 
   @Test
+  @Ignore
+  // There are some cases in alpha content where we don't have raw latex but we do have
+  // svg image, so can we use that image without raw latex?
```

Comment:
earlier we show only when both raw latex and SVG image is available in data, but as per our last discussion and from the alpha data, there are cases where we might have raw latex or svg or both of them, so I had conditioned them here.